### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by htgoebel**
*  	More readable debug log for rights regexps.
* 	Speed up reading defined rights.

### CODE REVIEW

**Comment by htgoebel** [comment]
Note: I needed to up my first commit.

**Comment by htgoebel** [comment]
I fixed the flak8 issues now. Sorry for the noise of intermediate commits, I confused my branches.

**Comment by htgoebel** [comment]
Rebased on top of release 2.0 to solve merge-conflicts.

File: radicale/rights.py [hunk 1]
```diff
@@ -60,32 +60,24 @@ def load(configuration, logger):
...
-user:.+
-collection:%(login)s(/.*)?
-permission:rw
-[r]
-user:.+
-collection:
-permission:r
-    """}
+    "authenticated": {
+        'rw': {'user': '.+',
```
Comment (Unrud) [inline radicale/rights.py]: Why are you using ``'`` instead of ``"``?

File: radicale/rights.py [hunk 2]
```diff
@@ -121,28 +113,31 @@ def authorized(self, user, path, permission):
...
                 re_collection = re_collection.format(*user_match.groups())
                 # Emulate fullmatch
                 if re.match(r"(?:%s)\Z" % re_collection, sane_path):
-                    self.logger.debug("Section '%s' matches", section)
+                    self.logger.debug("  Section %r: matches %r",
+                                      section, re_collection)
                     return permission in regex.get(section, "permission")
                 else:
-                    self.logger.debug("Section '%s' does not match", section)
+                    self.logger.debug("  Section %r: "
```
Comment (Unrud) [inline radicale/rights.py]: In my opinion this is too much log output. I think it should log a successful match and if no match was found.

File: radicale/rights.py (hunk 1 above)
Comment (htgoebel) [inline radicale/rights.py]: No special reason. Both are the same in Python, so I typically don't care

File: radicale/rights.py (hunk 2 above)
Comment (htgoebel) [inline radicale/rights.py]: My change is mimicking the old behaviour:
Old:
```
Reading rights from file '/tmp/radicale/etc/radicale/rights'
Test if 'michael:michael' matches against '^admin$:.*' from section 'admin'
Test if 'michael:michael' matches against '.+$:^michael/(holidays|foobar)$' from section 'public'
Section 'public' does not match
Test if 'michael:michael' matches against '^(.+)-mobil?$:^michael/(calendar|shared-calendar|addr-handy)' from section 'mobile-calendars-rw'
Test if 'michael:michael' matches against '^(.+)-mobil?$:^michael/?$' from section 'mobile-darf-eigene-collections-lesen'
Test if 'michael:michael' matches against '^(.+)-mobil?$:.*' from section 'mobile-darf-sonst-nix'
Test if 'michael:michael' matches against '.+:^michael/.+' from section 'owner-write'
Section 'owner-write' does not match
```
New:
```
Reading rights from file '/tmp/radicale/etc/radicale/rights'
Testing user 'michael' for collection 'michael'
  Section 'admin': User does not match
  Section 'public': Collection does not match '^michael/(holidays|foobar)$'
  Section 'mobile-calendars-rw': User does not match
  Section 'mobile-darf-eigene-collections-lesen': User does not match
  Section 'mobile-darf-sonst-nix': User does not match
  Section 'owner-write': Collection does not match '^michael/.+'
```
You may notice that the old code did report every test, but not whether it matched or not.

**Comment by htgoebel** [comment]
Closing without any comment is bad style.

**Comment by Unrud** [comment]
> Closing without any comment is bad style.

Sorry, the reason closing was:

  * DEFINED_RIGHTS doesn't exist anymore.
  * The logging was changed in a different way.

**Pull request closed without merging**